noir-projects/aztec-nr/address-note/src/address_note.nr

```diff
@@ -8,6 +8,8 @@ use dep::aztec::{
 };
 
 global ADDRESS_NOTE_LEN: Field = 3;
+// ADDRESS_NOTE_LEN * 32 + 32(storage_slot as bytes) + 32(note_type_id as bytes)
+global ADDRESS_NOTE_BYTES_LEN: Field = 3 * 32 + 64;
```

Comment:
It seems quite likely that this will get rekt or be missing. Arithmetic over generics, the lord and saviour please cleanse me.

Reply:
I tried to generate it in the macro, but way too much magic and the global should be exposed anyway. What I think I can do in the meantime is perform the calculation and throw an error if the value is not what we expect, wdyt?

Reply:
Added an assertion telling the user to get rekt if they somehow miscalculate this value